datahub-actions/setup.py

```diff
@@ -29,6 +29,7 @@ def get_long_description():
 
     return description
 
+
 acryl_datahub_min_version = os.environ.get("ACRYL_DATAHUB_MIN_VERSION", "0.9.4")
 
 acryl_datahub_min_version = os.environ.get("ACRYL_DATAHUB_MIN_VERSION") or "0.10.3"
```

Comment:
Dont' we have to bump this actually? So we for sure have a version with `_resolve_element` and not `resolve_element`

Reply:
bumped it